### PR TITLE
adding r/place Tidbyt app

### PR DESCRIPTION
**Pull request by funkfinger**
Adding this again since I had a git user issue on my end. This is a Reddit r/place tidbit viewer that will show a random part of the final and cleaned r/place Reddit image. Below is an example of what to expect...

r_place_2022-doge

### CODE REVIEW

**Comment by austinrfnd** [comment]
I am super excited to get this accepted and in.

File: apps/redditrplace/reddit_r_place.star [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    # grab a coordinate from r/place image...
+    x = -1 * (random.number(WIDTH, 2000 - TILE_WIDTH))
+    y = -1 * (random.number(HEIGHT, 2000 - TILE_HEIGHT))
+
+    # randomized which way it scrolls...
+    scrollX = x - ((random.number(0, 2 * WIDTH)) - WIDTH)
+    scrollY = y - ((random.number(0, 2 * HEIGHT)) - HEIGHT)
+
+    # get a the final r/place image
+    url = "https://i.imgur.com/rzUhL4w.png"
```
Comment (betterengineering) [inline apps/redditrplace/reddit_r_place.star:30]: Could this image be cached?

Comment (funkfinger) [reply apps/redditrplace/reddit_r_place.star:30]: For sure! Wasn't sure how the backend handles it, but as it currently exists, it's a static image. Gone ahead and cached for 30 days

Comment (funkfinger) [reply apps/redditrplace/reddit_r_place.star:30]: adding another pull request to incorporate the cache...

**Pull request merged**